mypy-baseline.txt

```diff
@@ -514,6 +510,13 @@ posthog/hogql/test/test_resolver.py:0: error: Incompatible types in assignment (
 posthog/hogql/test/test_resolver.py:0: error: "TestResolver" has no attribute "snapshot"  [attr-defined]
 posthog/hogql/test/test_resolver.py:0: error: Incompatible types in assignment (expression has type "Expr", variable has type "SelectQuery")  [assignment]
 posthog/hogql/test/test_resolver.py:0: error: "TestResolver" has no attribute "snapshot"  [attr-defined]
+posthog/hogql/test/test_resolver.py:0: error: Item "None" of "JoinExpr | None" has no attribute "next_join"  [union-attr]
```

Comment:
IMO it doesn't make sense to assert in tests that these fields are not None, but I could do it.

Reply:
Yup that's fine. I agree that additional asserts wouldn't help code quality.